.github/workflows/build-deploy-pudl.yml

```diff
@@ -83,6 +85,11 @@ jobs:
       - name: Set up Cloud SDK
         uses: google-github-actions/setup-gcloud@v1
 
+      - name: Determine commit information
+        run: |-
```

Comment:
TIL about the [block chomping operator](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator).  Does GHA complain when there's an extra newline at the end here?